src/profile/components/editProfileForm/EditProfileForm.tsx

```diff
@@ -152,10 +152,13 @@ function EditProfileForm(props: Props) {
           city: props.profile.primaryAddress.city || '',
           primary: props.profile.primaryAddress.primary || true,
           countryCode: props.profile.primaryAddress.countryCode || 'FI',
+          __typename: props.profile.primaryAddress.__typename || 'AddressNode',
         },
         primaryPhone: {
           ...props.profile.primaryPhone,
+          primary: props.profile.primaryPhone.primary || true,
```

Comment:
A comment in the code explaining why this defaults to true could be helpful.

Reply:
Added comments to both primary address and phone.